src/Commands/TerminusCommand.php

```diff
@@ -2,34 +2,36 @@
 
 namespace Pantheon\Terminus\Commands;
 
+use Pantheon\Terminus\Config;
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
 use Symfony\Component\Console\Input\InputAwareInterface;
 use Robo\Contract\OutputAwareInterface;
 use Robo\Common\OutputAwareTrait;
 use Robo\Common\InputAwareTrait;
 use Terminus\Models\Auth;
 
-abstract class TerminusCommand implements InputAwareInterface, OutputAwareInterface
+abstract class TerminusCommand implements InputAwareInterface, LoggerAwareInterface, OutputAwareInterface
 {
     use InputAwareTrait;
+    use LoggerAwareTrait;
     use OutputAwareTrait;
 
     /**
-     * @var Config
-     */
-    protected $config;
-    /**
      * @var boolean True if the command requires the user to be logged in
      */
     protected $authorized = false;
+    /**
+     * @var Config
+     */
+    protected $config;
 
     /**
      * TerminusCommand constructor
-     *
-     * @param Config $config Terminus configuration object
      */
-    public function __construct($config)
+    public function __construct()
     {
-        $this->config = $config;
+        $this->config = new Config();
```

Comment:
Would using an ioc container make a difference? My brain starts to hurt when I think about service containers (and the yaml loading mess of Symfony's container makes it worse). If an upstream change in Robo would make it easier, we could submit a PR there too. For now, newing up an object in the constructor isn't the end of the world, but we should fight it like the plague before we take on too much tech debt.